spec/acceptance/suites/simp_env/00_simp_bolt_controller_spec.rb

```diff
@@ -0,0 +1,136 @@
+require 'spec_helper_bolt'
+
+test_name 'Install SIMP via Bolt'
+
+describe 'Install SIMP via Bolt' do
+
+  # Enable ssh login with passwords
+  hosts.each do |host|
+    it 'should enable ssh with passwords' do
+      on(host, 'sed -i "s/PasswordAuthentication no/PasswordAuthentication yes/g" /etc/ssh/sshd_config')
+      os = fact_on(host,'operatingsystemmajrelease')
+      if os.eql?('7')
+        on(host, 'systemctl restart sshd')
+      else os.eql?('6')
+        on(host, 'service sshd restart')
+      end
+    end
+  end
+
+  let(:run_cmd) {'runuser vagrant -c '}
+  let(:files_dir) { File.join(File.dirname(__FILE__), 'files') }
+  let(:skeleton_dir) { '/usr/share/simp/environment-skeleton' }
+
+  context 'on Bolt controller' do
+    # Install SIMP and Bolt
+    it 'should install SIMP and Bolt rpms' do
+      bolt_controller = only_host_with_role(hosts, 'boltserver')
+      on(bolt_controller, 'curl -s https://packagecloud.io/install/repositories/simp-project/6_X/script.rpm.sh | sudo bash')
```

Comment:
We don't typically use this script provided by packagecloud, because it doesn't work for OEL servers.  Instead we use the helpers in simp-beaker-helpers.

Reply:
Replaced packagecloud script with repo_helper.rb.